core/src/test/java/org/testcontainers/containers/wait/internal/InternalCommandPortListeningCheckTest.java

```diff
@@ -3,31 +3,36 @@
 import com.google.common.collect.ImmutableSet;
```

Comment:
could you please fix the indentation in this file? 
Also, IMO we can just use `nc` for this test to avoid creating a config for nginx

Reply:
oh yes, sorry, will fix the indentation.

I had a problem with netcat and the infinispan container before, it's not installed. I could imagine other containers not supporting it, too. In fact I thought this was the reason for the /proc/net/tcp solution here.

Reply:
ah, ok, I see 👍